### PR TITLE
Wip/ss/add ".lsp" file to project

### PR DESCRIPTION
**Pull request by shuang6**

### CODE REVIEW

**Comment by JD-Howard** [comment]
These tests seem to be working for me.

Unrelated, but bringing it to your attention. A while back we ran into an issues (#36) with spaces in PRJ names. It might not be a bad idea to create a test to insure this code remains intact:
https://github.com/Autodesk-AutoCAD/AutoLispExt/blob/47985cdca3cda633312e2fc9b9d923c34769d98e/extension/src/project/openProject.ts#L67-L73

File: extension/src/test/suite/project.test.ts [hunk 1]
```diff
@@ -1,46 +1,52 @@
...
+import { AddFile2Project } from '../../project/addFile2Project';
+import { SaveProject } from '../../project/saveProject';
+import { excludeFromProject } from '../../project/excludefile';
+import * as fs from	'fs-extra';
+import { RefreshProject } from '../../project/refreshProject';
+import { getTreeItemTitle } from '../../project/projectutil';
+import { openLspFile } from '../../project/openLspFile';
+import { hasFileWithSameName } from '../../project/projectTree';
 
+var sinon = require("sinon");
```
Comment (Sen-real) [inline extension/src/test/suite/project.test.ts:15]: is it used?

**Review by Sen-real — COMMENTED** [review]
Please resolve the conflicts, and fill in the Objective, Abstractions, etc.

File: extension/src/test/suite/project.test.ts (hunk 1 above)
Comment (shuang6) [reply extension/src/test/suite/project.test.ts:15]: no,  i will delete it.

**Comment by nigma456** [comment]
Will submit PR with another new branch.

**Pull request closed without merging**